src/items/equipment/psychic_strike_node_advanced.json

```diff
@@ -0,0 +1,101 @@
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "magitech",
+    "abilityMods": {
+      "parts": []
+    },
+    "attributes": {
+      "ac": {
+        "value": ""
+      },
+      "customBuilt": false,
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": ""
+      },
+      "hp": {
+        "max": "",
+        "value": 19
+      },
+      "size": "medium",
+      "sturdy": false
+    },
+    "attuned": false,
+    "bulk": "",
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": []
+    },
+    "critical": {
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<table><tbody><tr><td><strong>Model</strong></td><td><strong>Level</strong></td><td><strong>Price</strong></td><td><strong>Damage</strong></td><th></th></tr><tr><td>Basic</td><td>4</td><td>2,100</td><td>2d8</td><th></th></tr><tr><td>Standard</td><td>9</td><td>13,500</td><td>5d8</td><th></th></tr><tr><td>Advanced</td><td>14</td><td>73,000</td><td>9d8</td><th></th></tr><tr><td>Elite</td><td>19</td><td>557,000</td><td>14d8</td><th></th></tr></tbody></table><p>A psychic strike node gradually accumulates your stray thoughts and releases them in an offensive burst of mental energy when activated as a standard action. The burst is a mind-affecting effect that affects all other creatures within 10 feet of you, dealing an amount of untyped damage based on the model. Affected creatures who succeed at a Will save (DC 10 + half your level + your choice of your Constitution modifier, Intelligence modifier, Wisdom modifier, or Charisma modifier, chosen when you install this augmentation) take half damage and negate any additional effects from the burst (below). Once you’ve activated the node, you can’t do so again until you’ve rested for 10 minutes to regain Stamina Points, though you can spend 1 Resolve Point to recharge it immediately.</p><p>If you have two psychic strike nodes of the same model (which requires having the ability to install two augmentations in your brain system), the second node doesn’t grant additional uses of the mental burst. Instead, you activate the nodes as a pair, augmenting the mental burst in one of the following ways, chosen each time you activate the augmentations: you increase the damage dice to d10s, the burst affects all creatures within 20 feet of you, you can exclude up to three creatures in the burst’s area to not affect, or you cause any creature who fails their Will saving throw to also become @UUID[Compendium.sfrpg.conditions.ZacF0l6IwqBwToTJ]{sickened} for 1d3 rounds.</p>"
+    },
+    "equippable": true,
+    "equipped": false,
+    "identified": true,
+    "level": 14,
+    "modifiers": [
+      {
+        "_id": "7b217b75-2573-4191-a101-d50ef3cc5f64",
+        "name": "Deception node",
```

Comment:
Looks like an incorrect duplication. Can add a damage entry too. The other marks need fixing too.

Reply:
Not sure what you mean by incorrect duplication. This augment can be gained twice with a few races, and the modifier is different for two augments. Perhaps a name change.
As for damage. Augments currently do not have a damage function.

Reply:
This modifier seems to have no relevance to this item, and seems to be for an entirely different item